frontend/setup/rate-limiter.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2025 The Linux Foundation and each contributor.
+// SPDX-License-Identifier: MIT
+
+import type { RateLimiterConfig } from '../server/types/rate-limiter';
+
+/**
+ * Rate limiter configuration
+ *
+ * Configuration for rate limiting rules here.
+ * Rules are evaluated in order; the first matching rule wins.
+ */
+const rateLimiterConfig: RateLimiterConfig = {
+  // Enable or disable the rate limiter globally
+  enabled: true,
+
+  // Default rate limit for routes not matching any specific rule
+  defaultLimit: {
+    maxRequests: parseInt(process.env.NUXT_RATE_LIMITER_MAX_REQUESTS || 200),
+    windowSeconds: parseInt(process.env.NUXT_RATE_LIMITER_WINDOW_SECONDS || 60),
+  },
+
+  // Secret used for hashing IPs for GDPR compliance
+  secret: process.env.NUXT_RATE_LIMITER_SECRET || 'this is a random secret string for hashing ips',
+
+  // Redis database number to use for rate limiting
+  redisDatabase: parseInt(process.env.NUXT_RATE_LIMITER_REDIS_DB, 10),
+
+  // Route-specific rules
+  rules: [
+    {
+      route: '/api/auth/*',
+      methods: ['POST'],
+      maxRequests: 5,
+      windowSeconds: 60, // 5 login attempts per minute
+    },
+    {
```

Comment:
can you add  /api/community/[slug].post.ts also here to disable rate limiting. Its a webhook and it receives quite a lot of data from external app so we dont want to have it rate limited

Reply:
Good catch, neither Joana or I thought of this. I implemented exclusion rules and added those two paths there.